protocol-designer/src/file-data/__tests__/createFile.test.ts

```diff
@@ -101,6 +101,29 @@ describe('createFile selector', () => {
   })
 
   it('should return a valid Python protocol file', () => {
+    // The labware in the fixtures have namespace "fixture", which makes them
+    // custom labware. Change their namespace to "opentrons" so that they're
+    // treated as standard labware.
+    const labwareEntitiesOpentrons = {
+      ...labwareEntities,
+      tiprackId: {
+        ...labwareEntities['tiprackId'],
```

Comment:
ah lint prefers this way for all of these:

```suggestion
        ...labwareEntities.tiprackId,
```

Reply:
Yeah. I think lint is wrong here because `labwareEntities['tiprackId']` is a key in a map, not a property of an object, but I'll change this just to make lint happy.